tslint.json

```diff
@@ -103,7 +103,6 @@
         "variable-declaration": "nospace"
       }
     ],
-    "typeof-compare": true,
```

Comment:
the changes in this file were all related to deprecation warnings when I was running `yarn lint` locally.